apps/condo/domains/contact/utils/clientCard.tsx

```diff
@@ -136,12 +134,14 @@ const SearchByPhoneSelectOption = ({ phone, property, unitName, unitType, type,
     const prefix = unitType === BuildingUnitSubType.Parking ? ParkingMessage : ShortFlatMessage
     const unitNameMessage = unitName && ` ${prefix} ${unitName}`
 
-    const router = useRouter()
     const { isSmall } = useLayoutContext()
 
     const handleClick = useCallback(
-        () => router.push(`/phone/${phone}?tab=${getClientCardTabKey(get(property, 'id'), type, unitName, unitType)}`),
-        [phone, property, router, type, unitName, unitType])
+        () => window.open(
```

Comment:
You need to check if window isn't `undefined` due to SSR

Reply:
done